.github/workflows/integration.yml

```diff
@@ -55,8 +62,12 @@ jobs:
       - name: Set up Linux
         if: matrix.os == 'ubuntu-latest'
         run: |
-          sudo apt-get update
-          sudo apt-get install git gcc g++ unixodbc-dev  libsasl2-dev libxml2-dev libxslt-dev
+          ./.github/scripts/worker_install_linux_deps.sh
```

Comment:
I like the idea of putting this type of stuff in a script. But now that it's only one command, why use `|` and a new line?

Reply:
mostly for uniformity and reducing lines to change later